src/Main.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Arrays;
+
+public class Main {
+    public static void main(String[] args){
+
+        // HomeTask 1
+        int[] arr2 = {1,4,3,2};
+        Arrays.sort(arr2);
+
+        boolean isSorted = true;
+        for (int s = 1; s < arr2.length; s ++){
+            if (arr2[s] < arr2[s - 1]) {
+                isSorted = false;
+                break;
+            }
+        }
+        System.out.println(arr2.length);
+        for (int l = 0; l < arr2.length; l++){
+            System.out.print(arr2[l]);
+            if (l < arr2.length - 1){
+                System.out.print(" ");
+            }
+        }
+        System.out.println();
+        System.out.println(isSorted);
+
+
+        // HomeTask 2
+
+        //Implementation 1
+        String stringToReverse = "]gnirts siht esrever lliw I[";
+        StringBuilder reversedString = new StringBuilder(stringToReverse);
+        reversedString.reverse();
+        System.out.println("\n" + reversedString.toString());
+
+        //Implementation 2
+        String stringToReverse1 = "]2 gnirts siht esrever lliw I[";
+        StringBuilder reversedString2 = new StringBuilder();
+
+        for (int i = stringToReverse1.length() - 1; i>= 0; i--){
+            reversedString2.append(stringToReverse1.charAt(i));
+        }
+        System.out.println("\n" + reversedString2.toString());
+
+        //Implementation 3
```

Comment:
the easiest example for the third implementation is using for loop to traverse the string in reverse order and build the reversed string using concatenation.
For this you should create StringBuilder object to hold the reversed string, then use for loop to traverse the string in reversed order(from the last character to the first), then using the append method StringBuilder append the current character in each loop iteration